### PR TITLE
Feature/sky

### PR DESCRIPTION
**Pull request by Allen-Cherian**
Proto file updated as per the discussions.

### CODE REVIEW

File: sky.proto [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  google.protobuf.Timestamp time = 2;
+  string ID = 3;
+  optional string from = 4;
+  optional string to = 5;
+  optional int32 amount = 6;
+  optional string message = 7;
+  optional string bannerImage = 8;
+  optional string headingText = 9;
+  optional string cta = 10;
+
```
Comment (vyshnavsdeepak) [inline sky.proto:40]: unnecessary line breaks

File: sky.proto [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+message FindReq {
+
+}
+
+message FindRes {
+
+}
+
+
```
Comment (vyshnavsdeepak) [inline sky.proto:111]: unnecessary line breaks

**Comment by vyshnavsdeepak** [comment]
Shall we squash these commits?

**Comment by vyshnavsdeepak** [comment]
option go_package = "github.com/getfexr/protos";

Shall we also set go_package respecting the current repo url?

File: sky.proto [hunk 3]
```diff
@@ -0,0 +1,126 @@
+syntax = "proto3";
+
+package protos;
+
+option go_package = "github.com/getfexr/sky/protos";
```
Comment (vyshnavsdeepak) [inline sky.proto:5]: option go_package = "github.com/getfexr/protos";

**Review by n1dhn — APPROVED** [review]

**Pull request merged**